src/modules/apps/e2e/report/completedApps.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import { Box, Color } from 'ink'
+
+import { SpecReport } from '../../../../clients/Tester'
+import { FailedSpec } from './failedApps'
+import { AppId } from './appId'
+import { AppProps } from './index'
+
+
+const completedAppColors = (failedSpecs: SpecResult[]) => {
+  if (failedSpecs.length === 0) return { bgGreen: true, black: true }
+  return { bgRed: true, white: true }
+}
+
+interface SpecResult {
+  specName: string
+  specReport: SpecReport
+}
+
+export const Completed: React.FunctionComponent<AppProps> = ({ appId, specs }) => {
+  const failedSpecs = Object.keys(specs).reduce((acum, curSpecName) => {
+    if(specs[curSpecName].state !== 'passed') {
+      acum.push({ specName: curSpecName, specReport: specs[curSpecName] })
+    }
+
+    return acum
+  }, [] as SpecResult[])
+
+  const failed = failedSpecs.length > 0
```

Comment:
This could be passed as an argument to `completedAppColors`, then it could be:
```tsx
const completedAppColors = (failed: boolean) => {
  if (failed) {
    return { bgRed: true, white: true }
  }

  return { bgGreen: true, black: true }
}
```